components/sup/src/manager.rs

```diff
@@ -1006,18 +1006,12 @@ impl Manager {
                 break ShutdownMode::Departed;
             }
 
-            // This formulation is gross, but it doesn't seem to compile on Windows otherwise.
-            #[allow(clippy::match_bool)]
-            #[allow(clippy::single_match)]
             #[cfg(unix)]
```

Comment:
Out of curiosity, is there no equivalent signal in windows?  How does a windows service restart cleanly?

Reply:
There really is no windows equivilent. The closest thing to a ubiquitous "restart me" signal would be a Windows Service restart but that only applies to programs running as windows services that must implement a specific interface with `OnStart` and `OnStop` methods. Outside of services apps typically handle this in app specific ways.